docs/articles/documentation/using-testcafe/programming-interface/Runner.md

```diff
@@ -0,0 +1,223 @@
+String                                                                                              | The path to the browser executable.                                                            | Local browser
+`{path: String, cmd: String}`                                                                       | The path to the browser executable (`path`) with command line parameters (`cmd`).              | Local browser
+[BrowserConnection](/testcafe/documentation/using-testcafe/programming-interface/BrowserConnection/)| The remote browser connection.                                                                 | Remote browser
+
+You are free to mix different types of objects in one function call. The `browsers` function concatenates the settings when called several times.
+
+#### Examples
+
+##### Using browser aliases
+
+```js
+runner.browsers(["firefox", "chrome"]);
+```
+
+##### Using the path to the browser executable
+
+```js
+runner.browsers("C:\\Program Files\\Internet Explorer\\iexplore.exe");
+```
+
+##### The path to the browser executable with command line parameters
+
+```js
+runner.browsers({
+    path: "C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe",
+    cmd: "--new-window"
+});
+```
+
+##### The remote browser connection
+
+```js
+var createTestCafe = require('testcafe');
+var testCafe = await createTestCafe('localhost', 1337, 1338);
+
+var remoteConnection = testcafe.createBrowserConnection();
+
+// Outputs remoteConnection.url to visit it from the remote browser.
+console.log(remoteConnection.url);
+remoteConnection.on('ready', () => {
+    await testCafe
+        .createRunner()
+        .browsers(remoteConnection)
+        .run();
+});
+```
+
+### <a class="anchor" name="screenshots"></a>screenshots(path [, takeOnFails]) → this
+
+Enables the test runner to take screenshots of the tested webpages.
+
+Parameter                  | Type    | Description                                                                   | Default
+-------------------------- | ------- | ----------------------------------------------------------------------------- | -------
+`path`                     | String  | The path to which the screenshots will be saved.
+`takeOnFails` *(optional)* | Boolean | Specifies if screenshots should be taken automatically whenever a test fails. | `false`
+
+The `screenshots` function must be called in order to enable TestCafe to take screenshots whenever the `screenshot` action is called from test code.
+If the `screenshots` function is not called, the `screenshot` action is ignored.
+
+The `takeOnFails` parameter handles a separate scenario of capturing the webpage. Set it to `true` to make TestCafe take a screenshot whenever a test fails.
+
+#### Example
+
+```js
+runner.screenshots('/home/user/screenshots/', true);
+```
+
+### <a class="anchor" name="reporter"></a>reporter(name, outStream]) → this
```

Comment:
I know the purpose of anchors. I just wonder if we need explicit anchors here or Jekyll's markdown parser will do a job for us